zebra/zebra_rib.c

```diff
@@ -2648,23 +2649,48 @@ static void rib_sweep_table(struct route_table *table)
 	struct route_node *rn;
 	struct route_entry *re;
 	struct route_entry *next;
+	struct nexthop *nexthop;
 	int ret = 0;
 
-	if (table)
-		for (rn = route_top(table); rn; rn = srcdest_route_next(rn))
-			RNODE_FOREACH_RE_SAFE(rn, re, next)
-			{
-				if (CHECK_FLAG(re->status, ROUTE_ENTRY_REMOVED))
-					continue;
+	if (!table)
+		return;
 
-				if (re->type == ZEBRA_ROUTE_KERNEL
-				    && CHECK_FLAG(re->flags,
-						  ZEBRA_FLAG_SELFROUTE)) {
-					ret = rib_uninstall_kernel(rn, re);
-					if (!ret)
-						rib_delnode(rn, re);
-				}
-			}
+	for (rn = route_top(table); rn; rn = srcdest_route_next(rn)) {
+		RNODE_FOREACH_RE_SAFE(rn, re, next)
+		{
+			route_entry_dump(&rn->p, NULL, re);
```

Comment:
Maybe we need to guard this debug with `IS_ZEBRA_DEBUG_RIB`? Even without any debug statement zebra is being very verbose during startup.